builder/clique_builder.go

```diff
@@ -157,13 +157,13 @@ func (cb *CliqueBuilder) onChainHeadEvent(block *types.Block) error {
 		cb.slotCtxCancel()
 	}
 
-	slotCtx, slotCtxCancel := context.WithTimeout(context.Background(), 6*time.Second)
+	slotCtx, slotCtxCancel := context.WithTimeout(context.Background(), 12*time.Second)
```

Comment:
Why 12 seconds?
Extract "magic number: 12" to more descriptive variable / constant.